rpc/client.go

```diff
@@ -0,0 +1,90 @@
+package rpc // import "collectd.org/rpc"
+
+import (
+	"io"
+	"log"
+
+	"collectd.org/api"
+	pb "collectd.org/rpc/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+// client is a wrapper around pb.CollectdClient implementing Interface.
+type client struct {
+	ctx    context.Context
+	client pb.CollectdClient
```

Comment:
You could embed pb.CollectdClient and then call the RPC methods on a client object directly.